app/components/windows/EditStreamInfo.vue

```diff
@@ -70,7 +70,10 @@
           <h-form-group type="text" v-model="startTimeModel.date" :metadata="dateMetadata" />
           <h-form-group type="timer" v-model="startTimeModel.time" :metadata="timeMetadata" />
         </div>
-        <div v-if="selectedProfile">
+        <div
+          v-if="selectedProfile"
+          :class="{ profile: true, 'profile-default': selectedProfile.game === 'DEFAULT' }"
+        >
```

Comment:
cant you use the form to get the data here? i don't like the idea of making classes solely for the purposes of tests

Reply:
Unfortunately not to the best of my ability, due to not having the value anywhere in the DOM, nor being able to get it out via selectors reliably.